bulkaccessconditionoptions.md

```diff
@@ -0,0 +1,78 @@
+# BulkAccessConditionOptions Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/config/bulkaccessconditionoptions**   
+
+Provide access to the existent configurations for the bulk access control management. It returns the list of existent BulkAccessConfig.
+The BulkAccessConfig aggregates in a single place the configuration needed to the access section of the bulk access control management at the item and bitstream level.
+
+Please note that currently only a single repository wide configuration is supported, so this endpoint will list a single configuration.
+
+Status codes:
+* 200 OK - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. This endpoint is only accessible ot user that has administrative priviled over at least one community, collection or item limited to the resources that they admin.
+
+## Single bulkaccessconditionoptions-Form 
+**/api/config/bulkaccessconditionoptions/<:config-name>**
+
+*:config-name* the bulk access control management configuration name to retrieve
+
+Please note that currently only a single repository wide configuration is supported and it is named `default`
```

Comment:
Presumably this is configured somewhere on the backend?  Are these "default" settings defined in `access-conditions.xml`?  If so, can we mention that in these docs?